src/coprocessor/dag/expr/scalar_function.rs

```diff
@@ -108,6 +108,7 @@ impl ScalarFunc {
             | ScalarFuncSig::RegexpBinarySig
             | ScalarFuncSig::LeftShift
             | ScalarFuncSig::RightShift
+            | ScalarFuncSig::Pow
```

Comment:
You may need update `test_check_args` too

Reply:
I have fixed it.